content/pytorch/concepts/tensors/terms/zeros/zeros.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'ZEROS'
+Description: 'Initializing a new tensor with zeros.'
+Subjects:
+  - 'Data Science'
+  - 'Machine Learning'
+  - 'AI'
+Tags:
+  - 'AI'
+  - 'Data Structures'
+  - 'Arrays'
+  - 'Deep Learning'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'py-torch-for-classification'
+  - 'paths/machine-learning'
+---
+
+
+The `.zeros()` method returns a [Tensor](https://www.codecademy.com/resources/docs/pytorch/tensors) of a specified shape with zeros. Optionally, the output tensor, datatype, layout, device, and whether autograd records operations can be specified.
+
+## Syntax
+
+```py
+import torch
+
+torch.zeros(shape)
+
+# with defaults explicitly stated
+torch.zeros(shape, out=None, dtype=torch.float32, layout=torch.strided, device=torch.device('cpu'), requires_grad=False)
+```
+
+The shape is specified as a variable, tuple, or list of integers. Optional parameters are as follows:
+
+- `out`: The output Tensor, defaults to None.
+- `dtype`: Datatype (`torch.dtype`) of the zeros, defaults to the global default.
+- `layout`: Layout (`torch.layout`) of the output Tensor, defaults to `torch.strided`.
+- `device`: Device (`torch.device`) of the output Tensor, defaults to the global default.
+- `requires_grad`: Boolean of whether autograd will record operations on the output Tensor, defaults to False.
```

Comment:
Please refer to the link: https://pytorch.org/docs/stable/generated/torch.zeros.html

only the `layout` parameter has a default value other than global, all others have `None` as their default. Please reframe the parameter descriptions. Also add a bullet point for the `size` parameter